File: app/src/main/java/com/feriantes4dawin/feriavirtualmovil/data/network/UsuarioDataSource.java
package com.feriantes4dawin.feriavirtualmovil.data.network;

import androidx.lifecycle.LiveData;
import com.feriantes4dawin.feriavirtualmovil.data.models.ResultadoUsuario;

public interface UsuarioDataSource {

    LiveData<ResultadoUsuario> usuarioDescargado = null;

    void fetchDataUsuario(String email,String contrasena);
}

File: app/src/main/java/com/feriantes4dawin/feriavirtual/data/network/UsuarioDataSource.kt
package com.feriantes4dawin.feriavirtual.data.network

import androidx.lifecycle.LiveData
import com.feriantes4dawin.feriavirtual.data.network.responses.UsuarioResponse

interface UsuarioDataSource {

    var usuarioDescargado : LiveData<UsuarioResponse>

    suspend fun fetchDataUsuario(email:String,contrasena:String)
}